src/cascadia/TerminalSettingsModel/ActionAndArgs.cpp

```diff
@@ -52,6 +52,7 @@ static constexpr std::string_view SwitchToTabKey{ "switchToTab" };
 static constexpr std::string_view TabSearchKey{ "tabSearch" };
 static constexpr std::string_view ToggleAlwaysOnTopKey{ "toggleAlwaysOnTop" };
 static constexpr std::string_view ToggleCommandPaletteKey{ "commandPalette" };
+static constexpr std::string_view SaveTaskKey{ "saveTask" };
```

Comment:
Made an attempt at this in the commit below.  Not totally sure I got the names right.  Let me know if I should add this commit to the PR (totally understand if you want to update it yourself).

https://github.com/e82eric/terminal/commit/f5ceac9ee76b801d23a572c06da7656d03da1592

Reply:
Testing this I realized that I have an issue where you can save a snippet without a command line and a validation warning about the incorrect configs pops up after.  I think this commit would fix it (It may need to be localized)

https://github.com/e82eric/terminal/commit/d6873c17f5dde1fad0be4f7edd55a5ed3ef75cf6